src/ucp/core/ucp_ep.h

```diff
@@ -173,6 +173,15 @@ typedef struct ucp_ep_msg_config {
 } ucp_ep_msg_config_t;
 
 
+/*
+ * Inline thresholds with memtype mds
```

Comment:
Thresholds with and without non-host memory

(we can use this struct for rndv as well, for example)